### DISCUSSION TITLE
Firmware doesn't flash, screen goes black and never change

### DISCUSSION THREAD

**Discussion by bBuNaG1** [Jyers/Marlin#895] [Q&A] [answered]
Hi! I'm looking for UBL-BLTOUCH 10x10 firmware with Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN ACTIVATED, I already tried to compile the UBL-BLTouch-10x10 template but when I try to flash it stays black.

No problem flashing the precompiled ones, but i need to activate the z_min_probe...  due to this https://github.com/Jyers/Marlin/discussions/192

I have the  v4.2.2 mainboard and already updated the screen firmware

**Comment by Jyers** [comment, marked as answer]
Try renaming the file or reformatting the sd card, those usually solve issues like this.

**Comment by agalakhov** [comment]
This something happens if too many features are enabled. The compiler does not estimate needed stack space. Sometimes the stack is just bigger than the available memory.
